package.json

```diff
@@ -30,10 +30,8 @@
   "license": "MIT",
   "devDependencies": {
     "@babel/core": "^7.2.2",
-    "@babel/preset-env": "^7.2.3",
     "@vue/test-utils": "^1.0.0-beta.25",
     "babel-core": "^7.0.0-bridge.0",
```

Comment:
@yohodopo / @eddyerburgh Also updating to `babel-jest@24` should enable us to remove the dependency on the babel-core bridge, as `babel-jest` supports `babel@7` now

Reply:
I guess it's is because of our dependency on `ts-jest`. Now that ts-jest v24.0.0 is released, we no longer need this.